hazelcast/src/main/java/com/hazelcast/internal/diagnostics/DiagnosticsLogWriter.java

```diff
@@ -93,6 +93,13 @@ protected DiagnosticsLogWriter write(long i) {
         return this;
     }
 
+    protected DiagnosticsLogWriter writeDate(long epochTimeMs) {
+        date.setTime(epochTimeMs);
+        appendDateTime();
```

Comment:
`appendDateTime()` will reset `date` field's time to `System.currentTimeMillis()`. Is the intention here to execute `appendDateTime(epochTimeMs)` (instead of two statements above)?